pkg/daemon/config_drift_monitor_test.go

```diff
@@ -497,13 +532,34 @@ func (tc *configDriftMonitorTestCase) writeIgnitionConfig(t *testing.T, ignConfi
 	}
 
 	// Write systemd units the same way the MCD does.
-	if err := writeUnits(ignConfig.Systemd.Units, tc.systemdPath, true); err != nil {
+	if err := writeUnits(ignConfig.Systemd.Units, tc.rootPath, true); err != nil {
 		return fmt.Errorf("could not write systemd units: %w", err)
 	}
 
 	return nil
 }
 
+func createFalseSSHKeyDir(rootPath string, content []ign3types.SSHAuthorizedKey) error {
+	fpath := rootPath + "/home/core/.ssh/authorized_keys.d/ignition"
```

Comment:
question: Would it make sense to use `filepath.Join()` for this concatenation operation? I tend to prefer using the `filepath` helpers for any kind of path manipulations only because it handles certain edge-cases pretty well. For example, if called with something like: `filepath.Join("////", "/", "/home/core/.ssh")`, it will return `/home/core/.ssh`.

Reply:
agreed